proposals/description/fip_98.ts

```diff
@@ -0,0 +1,31 @@
+import { ProposalDescription } from '@custom-types/types';
+
+const fip_98: ProposalDescription = {
+  title: 'FIP-98: Purchase 10m VOLT',
+  commands: [
+    {
+      target: 'fei',
+      values: '0',
+      method: 'mint(address,uint256)',
+      arguments: ['{feiDAOTimelock}', '10170000000000000000000000'],
+      description: 'Mint $10.17M FEI'
+    },
+    {
+      target: 'fei',
+      values: '0',
+      method: 'approve(address,uint256)',
+      arguments: ['{voltFeiSwapContract}', '10170000000000000000000000'],
+      description: 'Approve 10.17m Fei to be spent by the otc contract on the timelocks behalf'
+    },
+    {
+      target: 'voltFeiSwapContract',
```

Comment:
Ideally Fei protocol is deploying its VOLT somewhere right?

Reply:
Where would Fei protocol deploy VOLT? In the uniswap v3 pair as an LP? VOLT is a productive asset on its own, you don't need to stake it to get yield.

Reply:
Once VOLT staking comes out, we can reassess this issue.